src/main/java/com/yahoo/sketches/theta/ConcurrentDirectThetaSketch.java

```diff
@@ -54,39 +54,44 @@
    * Returns a (fresh) estimation of the number of unique entries
    * @return a (fresh) estimation of the number of unique entries
    */
-  @Override public double getEstimationSnapshot() {
+  @Override
```

Comment:
This is probably a setting in your IDE, but the style in the library is for Override annotations to be on a separate line.  This makes the public/private modifier much easier to see as it stands out on its own line.  This should be part of Checkstyle, but isn't yet.